free-programming-books-es.md

```diff
@@ -58,20 +63,20 @@
 
 * [Algoritmos y Programación (Guía para docentes)](http://www.eduteka.org/pdfdir/AlgoritmosProgramacion.pdf) (PDF)
 * [Aprenda a programar como si estuviera en primero](http://www1.ceit.es/asignaturas/Informat1/ayudainf/aprendainf/Programar/Programar.pdf) (PDF)
-* [Apuntes de Algoritmos y Estructuras de Datos](https://146fd953-a-62cb3a1a-s-sites.googlegroups.com/site/prog3unlp/home/exp_algoritmos.pdf) (PDF)
+* [Apuntes de Algoritmos y Estructuras de Datos](https://146fd953-a-62cb3a1a-s-sites.googlegroups.com/site/prog3unlp/home/exp_algoritmos.pdf?attachauth=ANoY7cpP-tS9_7vAio7Zm7sr3kuK_rfKNFRjLCLT99k8BTxN1ZNUtYwkeTHPmCXHGmk3ruFmw6gXJxFI7eUElxP-6__Z28cUtYQDYg0HzZ104WMkc1i3TbQ2NMyQejkUvw1Xj5jY4tig4ghNyQhesMV2yogm33JkaNyVIQsKFu7J6mn__taPYsCcgzb5GwdhQECVJYrA3RBakA9jTXXWx6gA9p7i8hb_je8g9W3r35eN9AE0I9uGb8U%3D&attredirects=0) (PDF)
```

Comment:
How about using the Open Libra link for this https://openlibra.com/en/book/apuntes-de-algoritmos-y-estructuras-de-datos

Reply:
:+1: I'm looking at the rest. :smile_cat: